examples/iscsi-pd/iscsi-pd.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "iscsipd",
+    "kind": "Pod",
+    "apiVersion": "v1beta1",
```

Comment:
I was told to use v1b3 examples.  I assumed that was broadly applied.  I can correct my examples...